scripts/release.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-console */
+import fetch from 'node-fetch';
+
+const version = process.env.VERSION;
+
+const createTag = async () => {
+  try {
+    const payload = {
+      tag_name: `${process.env.PACKAGE}-v${version}`,
+      name: `${process.env.PACKAGE}-v${version}`,
+      body: `${process.env.PACKAGE} release version v${version}`,
+      draft: false,
+      prerelease: false,
+    };
+
+    const response = await fetch(
```

Comment:
I'm assuming we can iterate on this api call and include a zipped version of the formation build output.

Reply:
We should be able to. I have some time today to try this out and add a function that can actually do this for us. Here's a link that describes how to do it.

https://developer.github.com/v3/repos/releases/#upload-a-release-asset

Reply:
Looks like this will take longer to implement, but I do think this is possible. The only gotcha is zipping the files. Since formation and formation-react builds are different you will have to indicate what files are needed in the zip folder. I think formation is pretty straight forward but formation-react needs to be more defined. Overall I think this is possible and we shouldn't have a problem implementing this in the future.